dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -93,11 +96,17 @@ const actions = [
     key: "delete",
     value: true,
   },
+  {
+    name: "Quisby Compare",
```

Comment:
Let's just call this "Compare", because the name "Quisby" won't be meaningful to most users and it's just complicating the action popup.

Reply:
Gotcha